ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -999,3 +999,47 @@ def test_qr(
         atol=1e-2,
         ground_truth_backend=frontend,
     )
+
+# sqrtm
+@handle_frontend_test(
+    fn_tree="tensorflow.linalg.sqrtm",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_value=-100,
+        max_value=100,
+        shape=helpers.ints(min_value=1, max_value=20).map(lambda x: tuple([x, x])),
+    ).filter(
+        lambda x: "bfloat16" not in x[0]
+        and np.linalg.cond(x[1][0]) < 1 / sys.float_info.epsilon
+        and np.linalg.det(np.asarray(x[1][0])) != 0
+    ),
+    adjoint=st.booleans(),
+    test_with_out=st.just(False),
+)
+def test_sqrtm(
```

Comment:
this is not the correct way to write frontend tests. You should make use of the `helpers.test_frontend_function` which will handle the value assertion logic etc. You can look at the way the other tests have been written as an inspiration 🙂